src/test/java/core/basesyntax/strategy/ActivitiesStrategyImpTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.dao.ProductDaoImp;
+import core.basesyntax.model.ProductTransaction;
+import core.basesyntax.strategy.activities.ActivitiesHandler;
+import core.basesyntax.strategy.activities.AddActivitiesHandler;
+import core.basesyntax.strategy.activities.SubstractActivitiesHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ActivitiesStrategyImpTest {
+    private static Map<String, Integer> storage;

```

Comment:
i think it's better to use class Storage

Reply:
Storage is like a real database, so we need test without changing the real DB